spec/system/claims/support/schools/claims/change_claim_on_check_page_spec.rb

```diff
@@ -216,21 +197,25 @@ def when_i_add_training_hours(hours)
   end
 
   def then_i_expect_the_training_hours_to_be_selected(hours)
-    find("#claims-support-claim-mentor-training-form-hours-completed-#{hours}-field").checked?
+    if hours.to_i == 20
+      find("#claims-add-claim-wizard-mentor-training-step-hours-to-claim-maximum-field").checked?
+    else
+      find("#claims-add-claim-wizard-mentor-training-step-hours-to-claim-custom-field").checked?
+    end
   end
 
   def then_i_expect_the_training_hours_for(hours, mentor)
     expect(page).to have_content("Hours of training for #{mentor.full_name}")
-    find("#claims-support-claim-mentor-training-form-hours-completed-#{hours}-field").checked?
+    then_i_expect_the_training_hours_to_be_selected(hours)
   end
 
-  def then_i_check_my_answers(claim_record, provider, mentors, mentor_hours)
+  def then_i_check_my_answers(provider, mentors, mentor_hours)
     expect(page).to have_content("Check your answers")
 
     within("dl.govuk-summary-list:nth(1)") do
       within(".govuk-summary-list__row:nth(1)") do
         expect(page).to have_content("Academic year")
-        expect(page).to have_content(claim.academic_year_name)
+        expect(page).to have_content(@claim_window.academic_year.name)
```

Comment:
Similar to a previous comment 😁

```suggestion
        expect(page).to have_content(@claim_window.academic_year_name)
```